src/transformers/models/wav2vec2/modeling_wav2vec2.py

```diff
@@ -67,50 +67,43 @@ def _compute_mask_indices(
     Adapted from `fairseq's data_utils.py
     <https://github.com/pytorch/fairseq/blob/e0788f7007a8473a76db573985031f3c94201e79/fairseq/data/data_utils.py#L376>`__.
     """
+    torch_device = "cuda" if torch.cuda.is_available() else "cpu"
```

Comment:
Actually, people might use "cpu" even when "cuda" is available. Could we instead add a new arg to the function `device` and set it to `device=self.device` (to the model's device) when calling the function?

Reply:
Ok I will create a `device` argument to pass the device type to the function. Can I pass the [`hidden_states.device`](https://github.com/huggingface/transformers/blob/11655fafdd42eb56ad94e09ecd84d4dc2d1041ae/src/transformers/models/wav2vec2/modeling_wav2vec2.py#L842)  as the `device` argument when the `_compute_mask_indices` method is called inside the forward function?